modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/snapshot/programs/SnapshotProcessor.scala

```diff
@@ -136,6 +147,7 @@ sealed abstract class SnapshotProcessor[F[_]: Async: KryoSerializer: SecurityPro
             tipsToRemove = tipsToRemove
           )
           .flatMap(_ => lastGlobalSnapshotStorage.set(globalSnapshot))
+          .flatMap(_ => collateralUpdateBalances(globalSnapshot))
```

Comment:
If you ignore returned values anyway then `flatTap`

Reply:
Changed implementation - this code doesn't exist anymore.